tools/spotless/openremote-license-header.txt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright $YEAR, OpenRemote Inc.
```

Comment:
Many files are missing the header and it is quite some work to figure out the exact year files got added. We could also change it into a range so it is always correct. The Git history keeps track of all the changes anyhow.

```suggestion
 * Copyright 2015-$YEAR, OpenRemote Inc.
```

Reply:
I think a fixed year of last edit is fine, a range becomes more problematic unless you are trying to show the company origin year which is not what we need to convey here.